scripts/runTests.mjs

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { spawn } from 'node:child_process';
+
+try {
+  const argument = process.argv[2];
+
+  if (argument?.includes('filter')) {
+    spawn(`turbo run test ${argument}`, {
+      stdio: 'inherit',
+      shell: true,
+    });
+  } else if (argument) {
+    spawn(
+      `conc -g -n "CSS-workshop,Unit,A11y,React-workshop,e2e" -c "#8BCC2F,#D8598C,#539AD0,#D1B52A,white" "pnpm --filter=css-workshop run test --filter=${argument.toLowerCase()}" "pnpm --filter=@itwin/itwinui-react run test:unit ${argument}" "pnpm --filter "./testing/a11y" run test  --env componentName=${argument}" "pnpm --filter=react-workshop run test --spec="**/${argument}.*"" "pnpm --filter=e2e run test -- ${argument}"`,
```

Comment:
Would the best solution involve not bypassing the turbo pipeline or is running the necessary commands enough?

Reply:
I don't think we can *not* bypass the turbo pipeline if we want to pass args to bespoke test commands. But we can run the build ourselves in this script and still use turbo to run dependent build commands and cache outputs.

`css-workshop` needs `pnpm run build --filter=css-workshop`
`unit` doesn't need build
`a11y` needs `pnpm run build --filter=a11y`
`react-workshop` needs `pnpm run build --filter=react-workshop`
`e2e` needs `pnpm run build --filter=e2e`

It might be worth adding these as top-level scripts in `package.json`. For example:

```
"test:css": "pnpm run build --filter=css-workshop && pnpm --filter=css-workshop run test",
… etc
```

and then we can simply run `pnpm run test:css --filter=${argument.toLowerCase()}` inside this JS script.

Reply:
Alright sweet! i'll make that change